subprojects/docs/src/docs/userguide/upgrading_version_5.adoc

```diff
@@ -380,6 +380,24 @@ Internally, `AbstractArchiveTask` prefers the new properties and methods like `g
 
 If your plugin/build only uses these types (and does not extend them), nothing has changed.
 
+==== Archives fail on duplicates
+
+Until now archives defaulted to the `INHERIT` duplicates strategy, allowing the same path to exist multiple times in an archive.
+
+In Gradle 6.0 we are switching to `FAIL`, prohibiting duplicate files in archives.
+If you still want to allow them, you can be specify that explicitly:
+
+```
+task archive(type: Zip) {
+    duplicatesStrategy = DuplicatesStrategy.INCLUDE // allow duplicates
+    archiveName = 'archive.zip'
+    from 'src'
+}
+```
+
+*Note* that `Copy` and `Sync` tasks are unaffected: they still use the `INHERIT` duplicates strategy as default.
```

Comment:
```suggestion
*Note* that `Copy` and `Sync` tasks are unaffected: they still use the `INCLUDE` duplicates strategy as default.
```

Reply:
Meaning that inherit will delegate to include?
I.e. https://github.com/gradle/gradle/blob/623062702ba0bd809a0393cd53bed37f3ff9e58e/subprojects/core/src/main/java/org/gradle/api/internal/file/copy/DefaultCopySpec.java#L75